ste/mgr-JobMgr.go

```diff
@@ -961,7 +964,14 @@ func (jm *jobMgr) scheduleJobParts() {
 				go jm.poolSizer()
 				startedPoolSizer = true
 			}
-			jobPart.ScheduleTransfers(jm.Context())
+			getCredential := true
+			if jm.credential != nil {
+				getCredential = false
+			}
+			jobPart.ScheduleTransfers(jm.Context(), getCredential, jm.credential)
```

Comment:
I'm confused here, shouldnt we get the credential before scheduling transfers?

Reply:
for minimal code changes in the first job part (and its corresponding scheduleTransfers call) we store that credential retrieved when the pipeline is created which is then used for other job parts.

Reply:
So basically JobMgr and JobParts will have separate creds?

Reply:
JobMgr will store the cred after the first jobPart does its first scheduleTransfers call and then pass it to the next jobParts